api/goldens/aptos_api__tests__simulation_test__test_simulate_transaction_with_valid_signature.json

```diff
@@ -0,0 +1,5 @@
+{
+  "message": "Simulated transactions must have a non-valid signature",
```

Comment:
Did you wanted to say must have valid signature? Otherwise the sentence would be weird.

Reply:
Actually, it should not be valid, because a malicious actor could execute the transaction without user's knowledge. I think we want to avoid the need for signature at all.